internal/services/dns_firewall/resource_test.go

```diff
@@ -1,14 +1,46 @@
 package dns_firewall_test
 
 import (
+	"context"
+	"fmt"
 	"os"
 	"testing"
 
+	cfv1 "github.com/cloudflare/cloudflare-go"
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/acctest"
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/utils"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 )
 
+func init() {
+	resource.AddTestSweepers("cloudflare_dns_firewall", &resource.Sweeper{
+		Name: "cloudflare_dns_firewall",
+		F: func(region string) error {
+			accountID := os.Getenv("CLOUDFLARE_ACCOUNT_ID")
+			ctx := context.Background()
+
+			client, err := acctest.SharedV1Client() // TODO(terraform): replace with SharedV2Clent
+			if err != nil {
+				return fmt.Errorf("error establishing client: %w", err)
+			}
+
+			clusters, err := client.ListDNSFirewallClusters(ctx, cfv1.AccountIdentifier(accountID), cfv1.ListDNSFirewallClustersParams{})
```

Comment:
i'd recommend using the newer SDK here to avoid the pains with the older library.

Reply:
I gave that a try, please let me know how it looks!